.gitignore

```diff
@@ -1,3 +1,3 @@
 *.pyc
-
+__pycache__/
```

Comment:
This file is auto-generated with re-rendering and you should not modify it directly.

Reply:
ok, I haven't re-rendered in a while and I thought it was weird that they wouldn't exclude cache files.

Reply:
It is not a python package so that directory should not appear in a feedstock. However, if you believe we really need that you can send a PR modifying https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/feedstock_content/.gitignore